author/iso/topics/markup.adoc

```diff
@@ -298,16 +298,29 @@ and definitions`:
 
 ....
 
-== Terms and definitions
-
-...
-
 [heading=terms and definitions]
 == Terms related to comedy
 
 ....
 ----
 
+Vocabulary documents usually do not contain a clause for Symbols and Abbreviated terms. 
+Preferably, those are included in term entries as preferred or admitted terms.
+Alternatively they can be listed in an annex:
+
+[source,adoc]
+----
+:docsubtype: vocabulary
+
+....
+
+[appendix]
+== {blank}
```

Comment:
@opoudjis why do we need a blank appendix?

Reply:
We don't, really. I just don't know what a symbols clause within an appendix would be called.

Reply:
"Symbols and abbreviated terms". I'll fix this.